### ISSUE TITLE
train.py backbone=resnet/mobilenet

### ISSUE THREAD

**Issue by 100sby** [jfzhang95/pytorch-deeplab-xception#73]
Thanks for your code.But when i traning with voc,and backbone=resnet,val miou is about 68.85%.And when  i used backbone=mobilenet,val miou is about 56.78%.It's different with your miou in val at "READ ME" .I set batch_size=4,epoches=50.So should i set batch_size=16 or more??

**Comment by jfzhang95** [comment]
Hi, you may try to freeze bn layer during training. Reported models were trained without updating bn weights.

**Comment by 100sby** [comment]
> Hi, you may try to freeze bn layer during training. Reported models were trained without updating bn weights.

OK,thank you!I will try.

**Comment by dfan** [comment]
According to the paper, not training batch normalization weights should lead to strictly worse results
